web/src/graph-viewer.ts

```diff
@@ -170,6 +244,29 @@ export class GraphViewer extends LitElement {
   }
 }
 
+@customElement("packages-outline")
+export class PackagesOutline extends LitElement {
+  // TODO Eventually, this should be a collection of Pkg and contain actual details.
+  @property({ attribute: false }) packages: Array<string> = ["hello"];
```

Comment:
Shouldn't it be empty?

```suggestion
  @property({ attribute: false }) packages: Array<string> = [];
```